DirectProgramming/DPC++FPGA/Tutorials/DesignPatterns/loop_carried_dependency/sample.json

```diff
@@ -13,19 +13,23 @@
       {
         "id": "fpga_emu",
         "steps": [
+          "dpcpp --version",
+          "cd ../../..",
           "mkdir build",
           "cd build",
-          "cmake ..",
+          "cmake ../Tutorials/DesignPatterns/loop_carried_dependency",
```

Comment:
./Tutorials/..., not ../Tutorials/...
(comment applies in multiple places)

Reply:
the first "cd ../../.." places us in DPC++FPGA/, then we cd into DPC++FPGA/build, so we have to go back up 1 level before Tutorials/...

Reply:
Oops, you're right!